pkg/image/utilities/utilities.go

```diff
@@ -677,7 +676,6 @@ Get-Content -Force -Path $image_list | ForEach-Object {
 $script_name = $MyInvocation.InvocationName
 $image_list = "rancher-windows-images.txt"
 $images = "rancher-windows-images.tar.gz"
-$os_release_id = $(Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\' | Select-Object -ExpandProperty ReleaseId)
 
 $help = $false
 
```

Comment:
we should remove like 686 and 705-707 as the option won't have any effect

Reply:
Removed.